<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>date_to_date</class>
 <widget class="QWidget" name="date_to_date">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>162</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="begin_edit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="end_edit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="begin_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>25</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Begin date (dd/mm/yyyy)</string>
   </property>
  </widget>
  <widget class="QLabel" name="end_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>86</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>End date (dd/mm/yyyy)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="show_data_button">
   <property name="geometry">
    <rect>
     <x>189</x>
     <y>100</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Show data</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_date_button">
   <property name="geometry">
    <rect>
     <x>289</x>
     <y>100</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="instructions_edit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>181</width>
     <height>70</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
